apps/central-scan/backend/src/fujitsu_scanner.ts

```diff
@@ -82,9 +89,35 @@ export class FujitsuScanner implements BatchScanner {
     );
   }
 
+  async isImprinterAttached(): Promise<boolean> {
+    return new Promise((resolve) => {
+      const process = streamExecFile('scanimage', [
+        '-d',
+        'fujitsu',
+        '--endorser=yes',
+        '--format=jpeg',
+        '-n',
```

Comment:
We should probably do this with the other short options as well, but this one in particular I had to look up to make sure this was doing what I expected.
```suggestion
        '--dont-scan',
```